tests/conftest.py

```diff
@@ -117,17 +117,24 @@ def record_props(request, record_property):
 def pytest_runtest_logreport(report):
     """Send general test metrics to CloudWatch"""
     if report.when == "call":
-        dimensions = {
-            "test": report.nodeid,
-            "instance": global_props.instance,
-            "cpu_model": global_props.cpu_model,
-            "host_kernel": "linux-" + global_props.host_linux_version,
-        }
+        METRICS.set_dimensions(
+            {
+                "test": report.nodeid,
+                "instance": global_props.instance,
+                "cpu_model": global_props.cpu_model,
+                "host_kernel": "linux-" + global_props.host_linux_version,
+            },
+            # per host kernel
+            {"host_kernel": "linux-" + global_props.host_linux_version},
+            # per CPU
+            {"cpu_model": global_props.cpu_model},
+            # and global
+            {},
+        )
```

Comment:
It will show up only once in the logs, but as different datapoints in the metrics/dashboards. Like the record here: 

```
{
    "_aws": {
        "Timestamp": 1715087294604, 
        "CloudWatchMetrics": [
            {
                "Dimensions": [["test", "instance", "cpu_model", "host_kernel"], ["host_kernel"], ["cpu_model"], []], 
                "Metrics": [{"Name": "duration", "Unit": "Seconds"}, {"Name": "failed", "Unit": "Count"}], 
                "Namespace": "local"
            }
        ]
    }, 
    "duration": 5.381340845997329, 
    "failed": 0
}
```

Reply:
So size increases a bit, but not but much

Reply:
Ahh, I see, neat! Thanks for explaining :)